src/integration/java/io/pinecone/integration/controlPlane/pod/ConfigureIndexTest.java

```diff
@@ -83,7 +79,7 @@ public void scaleUpAndDown() {
             });
 
             // Scaling down
-            pod = new ConfigureIndexRequestSpecPod().replicas(3);
+            pod = new ConfigureIndexRequestSpecPod().replicas(1);
             spec = new ConfigureIndexRequestSpec().pod(pod);
             configureIndexRequest = new ConfigureIndexRequest().spec(spec);
             controlPlaneClient.configureIndex(indexName, configureIndexRequest);
```

Comment:
Not sure why I can't comment on line#98 but can we update it to the following:
`throw new PineconeException("Test failed: " + exception.getLocalizedMessage());`

Currently the error will show a hashcode and not the message itself.

Reply:
Line#94 *